README.md

```diff
@@ -1,3 +1,38 @@
 # URL Shortener
 
 made by [Jawwad](https://github.com/JawwadUddin) and [Max](https://github.com/Velocima)
+
```

Comment:
README is good but could have been a bit more detailed (RE purpose, technologies used etc) ? :)